tests/utils_tests.py

```diff
@@ -32,6 +32,9 @@
 import lsst.daf.base as dafBase
 import lsst.geom
 
+# Create an instance of random generator with fixed seed.
+rng = np.random.default_rng(1234)
```

Comment:
In my testing, this results in different random catalogs for each test if I randomize the order the tests are run with `pytest --random-order`. I suggest instead defining `rng` in the `setUp` method of each class that needs it, and passing that in to `makeDiaObjects`, `makeDiaSources`, and `makeDiaForcedSources`